discord-webhook.ts

```diff
@@ -149,18 +120,29 @@ async function fetchJobs() {
 	return result as { jobs: GitHubActionsJob[] }
 }
 
+async function createDescription(suite: string, targetText: string) {
+	const runUrl = await createRunUrl(suite)
+
+	/* eslint-disable no-irregular-whitespace */
+	return `
+:scroll: [Open](${runUrl})　　:zap: ${targetText}
```

Comment:
Thanks for these changes @sapphi-red! And great detail by having two lines instead of one for alignment.

I think we could use two spaces instead of one between the emoji and text. I know this sounds bad, but it looks a lot better in Discord (I have been doing this in announcements). If at some point Discord fixes spacing between emojis and text, we can remove it.

And maybe we could use `:vite:` instead of `:zap:`? So it is even more clear that the ref is about Vite?
```suggestion
:scroll:  [Open](${runUrl})　　:vite:  ${targetText}
```

Reply:
I ended up using `\u00A0`(`&nbsp;`) to avoid discord merging spaces. (It seems they do this only inside embed description.)

Also `:vite:` wasn't available here. (I did add the custom emoji but it seems not possible to use it here)

![image](https://user-images.githubusercontent.com/49056869/182016304-53a95c35-18ae-43ac-93ae-906820f949da.png)
The last message is the current PR's one.

Reply:
Thanks for trying these out! Looks freat